src/System.Runtime/tests/System/IntPtrTests.cs

```diff
@@ -121,6 +121,19 @@ public static void Equals(IntPtr ptr1, object obj, bool expected)
             Assert.Equal(ptr1.GetHashCode(), ptr1.GetHashCode());
         }
 
+        [Theory]
+        [MemberData(nameof(Equals_TestData))]
+        public static void Equals_NetCoreApp11(IntPtr ptr, object obj, bool expected)
```

Comment:
Having the netcoreapp11 specific testing in separate method does not make sense anymore. Could you please merge this test into the `Equals` above?